content/guides/http/java/jetty/client.md

```diff
@@ -0,0 +1,407 @@
+---
+title: "Client"
+date: 2019-02-04T00:35:17-08:00
+draft: false
+aliases: [/integrations/http/java/jetty/client]
+logo: /images/jetty-header-logo.png
+---
+
+- [Introduction](#introduction)
+- [Imports](#imports)
+    - [Dependency management](#dependency-management)
+- [Initializing the client](#initializing-the-client)
+- [Enabling observability](#enabling-observability)
+    - [Stats](#stats)
+    - [Traces](#traces)
+    - [Extracing observability](#extracting-observability)
+- [End-to-end example](#end-to-end-example)
+    - [Source code](#source-code)
+    - [Running it](#running-it)
+- [Examining your traces](#examining-your-traces)
+    - [All traces](#all-traces)
+    - [Existent page](#existent-page)
+    - [Non-existent page](#non-existent-page)
+- [Examining your stats](#examining-your-stats)
+- [References](#references)
+
+### Introduction
+
+The Jetty client integration has been instrumented with OpenCensus and it provides traces and stats.
+This integration was added in OpenCensus-Java version 0.19.1.
+
+### Imports
+
+To add the Jetty client integration, we'll perform the following import.
+
+{{<highlight Java>}}
+import io.opencensus.contrib.http.jetty.client.OcJettyHttpClient;
+{{</highlight>}}
+
+#### Dependency management
```

Comment:
what about gradle?

Reply:
Roger that, I've added:
* Gradle
* Buildr
* Ivy

for the minimum additions of *http-jetty-client and *http-util dependencies.